assets/control-plane/prometheus-rule.yaml

```diff
@@ -291,7 +291,7 @@ spec:
     rules:
     - alert: KubePersistentVolumeFillingUp
       annotations:
-        description: The PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is only {{ $value | humanizePercentage }} free.
+        description: The PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} on Cluster {{ $labels.cluster }} is only {{ $value | humanizePercentage }} free.
```

Comment:
/hold

this change would introduce confusion since the `cluster` label isn't always set.

Reply:
I proposed https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/887. Do we want to hold this up until then?

Reply:
Thanks! Yes, let's wait until we have more visibility.